src/hooks/useParaswapCallback.ts

```diff
@@ -172,10 +169,11 @@ export function useParaswapCallback(
           .then((response: TransactionResponse) => {
             const inputSymbol = trade.inputAmount.currency.symbol;
             const outputSymbol = trade.outputAmount.currency.symbol;
-            const inputAmount = formatTokenAmount(trade.inputAmount);
-            const outputAmount = formatTokenAmount(trade.outputAmount);
+            const inputAmount = Number(rate.srcAmount) / 10 ** rate.srcDecimals;
+            const outputAmount =
```

Comment:
Won't this cause the display to show numbers as in an in appropriate format if it's too large 

E.g 1.57373e10